packages/sanity/src/_internal/cli/actions/app/devAction.ts

```diff
@@ -1,30 +0,0 @@
-import {type CliCommandArguments, type CliCommandContext} from '@sanity/cli'
```

Comment:
We should keep this file and the split since in future the dev action will be different for studio and apps